strings/italian_replacement.json

```diff
@@ -1,68 +1,68 @@
 {
 
-    "italian": {
-        "I'm": "I'm-a",
-        "am": "am-a",
-        "and": "and-a",
-        "assistant": "goombah",
-        "assistants": "goombahs",
-        "baby": [
-            "bambino",
-            "little sausage roll"
-        ],
-        "bad": "molto male",
-        "bye": [
-            "ciao",
-            "arrivederci"
-        ],
-        "captain": "capitano",
-        "cheese": [
-            "parmesano",
-            "gorgonzola"
-        ],
-        "cook": "cook-a",
-        "could": "could-a",
-        "dad": "pappa",
-        "enemy": "friend-a",
-        "friend": "enemy-a",
-        "good": "molto bene",
-        "greytide": "curvisti",
-        "greytider": "curvisti",
-        "greytiders": "curvisti",
-        "hello": [
-            "ciao",
-            "buongiorno"
-        ],
-        "it's": "it's-a",
-        "make": "make-a",
-        "meat": [
-            "pepperoni",
-            "prosciutto"
-        ],
-        "mom": "mamma",
-        "my": "my-a",
-        "nuke": "spiciest-a meatball",
-        "op ": "greek ",
-        "operative": "greek",
-        "operatives": "greeks",
-        "ops": "greeks",
-        "sec ": "polizia ",
```

Comment:
Bring these back - its important to have those spaces as instead things like "flopped" will turn into "flgreekped"

Reply:
returned it as it was